test/ctype_internal_test.c

```diff
@@ -14,11 +14,10 @@
 #include <stdio.h>
 
 /*
- * Even though the VMS C RTL claims to be C99 compatible, it's not entirely
- * so far (C RTL version 8.4). Same applies to OSF. For the sake of these
- * tests, we therefore define our own.
+ * Even though the OSF RTL claims to be C99 compatible, it's not entirely
+ * so far, so we we therefore define our own.
```

Comment:
```suggestion
 * so far, so we therefore define our own.
```
I'd also drop either the second "so" or the "therefore"